pkg/ui/serve.go

```diff
@@ -30,23 +31,23 @@ func Listen() {
 	}
 }
 
-func apply(cr string) {
+func apply(cr string) error {
```

Comment:
If you return the error, you should at most log it at debug verbosity. The caller will be able to log it, and they may have some retry logic based on the error that will mean the Error verbosity log will be confusing.

Reply:
Will be fixed in a follow-up